src/components/Banner/Banner.tsx

```diff
@@ -1,6 +1,7 @@
 import clsx from 'clsx';
 import React, { ReactNode, useState } from 'react';
 import styles from './Banner.module.css';
+import type { IconName } from '../../util/allIcons';
```

Comment:
we could re-export this from the `Icon` component, but this doesn't seem bad to me

Reply:
actually no I change my mind